### PR TITLE
Ping 3380, PING-3364 Change opacity and text search domains

### PR DESCRIPTION
**Pull request by agit26milan**
### Summary by CodeRabbit

- Style: Updated UI styles in `TopicsChip.js` and `OtherProfile/index.js` for better user experience. The changes include adjustments to opacity, display, padding, color, and other style properties.
- Bug Fix: Removed a condition in `FeedScreen/Content.js` that could potentially alter the return value of the `getLineCount` function, ensuring consistent behavior.
- New Feature: Enhanced search functionality in `StringConstant/index.js` by updating the placeholder text from "Search Domains" to "Search Recent Articles", providing clearer instructions to users.

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f284ff34e506cdb58a92b49dc6debd96a3e1f56e and 5c7aff2aa63a1c7ca5492c3010ed5aabdb2653b3 commits.
</details>

<details>
<summary>Files selected (4)</summary>

* src/components/TopicsChip/TopicsChip.js (1)
* src/screens/FeedScreen/Content.js (1)
* src/screens/OtherProfile/index.js (1)
* src/utils/string/StringConstant/index.js (1)
</details>






---

File: src/screens/FeedScreen/Content.js [hunk 1]
```diff
@@ -71,9 +71,6 @@ const Content = ({
 
       return result >= 0 ? result : 0;
     }
-    if (getCommentLength(item.latest_reactions.comment) > 0) {
-      return Math.floor(layoutHeight / lineHeight);
-    }
     return Math.round(layoutHeight / lineHeight);
```
Comment (bastian-purrer) [inline src/screens/FeedScreen/Content.js:74]: Image description   CodeRabbit

The removal of the condition that checks if a comment's latest reactions length is greater than zero could potentially lead to unexpected behavior. If `item.latest_reactions.comment` is expected to be an array, and it's empty or undefined, the function will now return `Math.round(layoutHeight / lineHeight)`, which might not be the intended behavior. Please ensure this change aligns with your application's requirements.
```diff
-    return Math.round(layoutHeight / lineHeight);
+    if (getCommentLength(item.latest_reactions.comment) > 0) {
+      return Math.floor(layoutHeight / lineHeight);
+    }
+    return Math.round(layoutHeight / lineHeight);
```

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between f284ff34e506cdb58a92b49dc6debd96a3e1f56e and 5c7aff2aa63a1c7ca5492c3010ed5aabdb2653b3 commits.
</details>

<details>
<summary>Files selected (4)</summary>

* src/components/TopicsChip/TopicsChip.js (1)
* src/screens/FeedScreen/Content.js (1)
* src/screens/OtherProfile/index.js (1)
* src/utils/string/StringConstant/index.js (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (2)</summary>

* src/components/TopicsChip/TopicsChip.js
* src/utils/string/StringConstant/index.js

</details>

<details>
<summary>Review comments generated (2)</summary>

* Review: 1
* LGTM: 1

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

File: src/screens/FeedScreen/Content.js (hunk 1 above)
Comment (agit26milan) [reply src/screens/FeedScreen/Content.js]: getCommentLength always return number, so it should not be cannot undefined

**Pull request merged**